docs/UserGuide.md

```diff
@@ -265,7 +265,7 @@ Update the details of a player, lineup or schedule.
 
 **To edit a player:**
 
-Format: `edit P/NAME [n/NAME] [p/PHONE_NUMBER] [w/WEIGHT] [h/HEIGHT] [j/JERSEY_NUMBER] [T/TAGS]`
+Format: `edit P/NAME [n/NAME] [p/PHONE_NUMBER] [w/WEIGHT] [h/HEIGHT] [j/JERSEY_NUMBER] [t/TAG]…​`
```

Comment:
closes #237, #235

Reply:
Thanks for helping to link all the issues!!! (I just realized how to do it correctly just now 😅